clis/teliod/src/main.rs

```diff
@@ -63,17 +91,35 @@ enum TeliodError {
     DaemonIsRunning,
     #[error("NotificationCenter failure: {0}")]
     NotificationCenter(#[from] nc::Error),
+    #[error(transparent)]
+    CoreApiError(#[from] ApiError),
+    #[error(transparent)]
+    DeviceError(#[from] DeviceError),
+    #[error("Unable to update machine due to Error: {0}")]
+    UpdateMachineError(StatusCode),
+    #[error("Max retries exceeding to update machine")]
+    UpdateMachineTimeout,
+    #[error(transparent)]
+    BackoffBoundsError(#[from] BackoffError),
+    #[error("No local data directory found")]
+    NoDataLocalDir,
 }
 
 #[tokio::main]
+#[allow(large_futures)]
 async fn main() -> Result<(), TeliodError> {
     match Cmd::parse() {
         Cmd::Daemon { config_path } => {
             if DaemonSocket::get_ipc_socket_path()?.exists() {
                 Err(TeliodError::DaemonIsRunning)
             } else {
-                let file = File::open(config_path)?;
-                let config: TeliodDaemonConfig = serde_json::from_reader(file)?;
+                let file = File::open(&config_path)?;
+                let mut config: TeliodDaemonConfig = serde_json::from_reader(file)?;
+                let token = std::env::var("NORD_TOKEN").ok();
+                if let Some(t) = token {
+                    debug!("Overriding token from env");
```

Comment:
The idea was so that in development someone doesn't accidentally end up committing their token. I mean, in that sense it would be better to have an option to load it from the env.